src/main/java/de/tum/cit/aet/artemis/atlas/domain/profile/CourseLearnerProfile.java

```diff
@@ -0,0 +1,39 @@
+package de.tum.cit.aet.artemis.atlas.domain.profile;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+import de.tum.cit.aet.artemis.core.domain.Course;
+import de.tum.cit.aet.artemis.core.domain.DomainObject;
+
+@Entity
+@Table(name = "course_learner_profile")
+public class CourseLearnerProfile extends DomainObject {
+
+    @ManyToOne
+    @JoinColumn(name = "learner_profile_id")
+    private LearnerProfile learnerProfile;
+
+    @OneToOne
```

Comment:
```suggestion
    @OneToOne(fetch = FetchType.LAZY)
```
We don't want to always load the course alongside the CourseLearnerProfile, do we?